sycl/test/on-device/lit.cfg.py

```diff
@@ -178,6 +179,7 @@ def getDeviceCount(device_type, be = backend):
 config.substitutions.append( ('%level_zero_options', level_zero_options) )
 
 sp = subprocess.getstatusoutput(config.clang + ' -fsycl  ' + check_l0_file + level_zero_options)
+Path(check_l0_file).unlink(missing_ok=True)
```

Comment:
What is this?

Reply:
A file that is created to check if a level_zero configuration is present, and which is not cleaned so lit was trying to run it as a test.